go.mod

```diff
@@ -1,112 +1,113 @@
 module github.com/openshift/router
 
-go 1.18
+go 1.19
 
 require (
 	github.com/MakeNowJust/heredoc v0.0.0-20171113091838-e9091a26100e
 	github.com/bcicen/go-haproxy v0.0.0-20180203142132-ff5824fe38be
 	github.com/cockroachdb/cmux v0.0.0-20170110192607-30d10be49292
 	github.com/fortytw2/leaktest v1.3.0
-	github.com/fsnotify/fsnotify v1.4.9
+	github.com/fsnotify/fsnotify v1.6.0
 	github.com/gocarina/gocsv v0.0.0-20190927101021-3ecffd272576
-	github.com/google/go-cmp v0.5.8
+	github.com/google/go-cmp v0.5.9
 	github.com/haproxytech/config-parser/v4 v4.0.0-rc1
-	github.com/openshift/api v0.0.0-20211201130627-34f305c3af47
-	github.com/openshift/client-go v0.0.0-20201214125552-e615e336eb49
-	github.com/openshift/library-go v0.0.0-20201223214116-830765adf874
-	github.com/prometheus/client_golang v1.12.1
-	github.com/prometheus/client_model v0.2.0
-	github.com/prometheus/common v0.32.1
-	github.com/spf13/cobra v1.4.0
+	github.com/openshift/api v0.0.0-20230120195050-6ba31fa438f2
```

Comment:
Do you understand why updating core deps (k8s.io/api, k8s.io/apimachinery, k8s.io/apiserver, and k8s.io/client-go) would bump openshift/api?

Do you have your go mod update runes? If so, it would be handy to enumerate those in the PR description.

Reply:
/hold

I just want to confirm our update method using `go get`.

In the PR description you mention `go get -u` whereas in https://github.com/openshift/cluster-dns-operator/pull/356 we're just using `go get`. Do we want to update (i.e., `-u`) transitive dependencies too? Did 356 also use `go get -u`?

Reply:
Good question. Should we update transitive dependencies? I noticed Miciah had used `-u` in one of his commands, so I went back and make sure my PRs used it, starting with this router PR. I didn't use `-u` for cluster-dns-operator yet.

Is there a reason we shouldn't update transitive dependencies?

Reply:
I updated https://github.com/openshift/cluster-dns-operator/pull/356 to use `-u`. Here's the [diff for 356 ](https://github.com/openshift/cluster-dns-operator/compare/4adaeabf5ab59f9097ff4175230a1a7c260debbe..a937e200b9d632aa294aa33ade67e4ba36e47f94)

Reply:
https://github.com/openshift/router/pull/451#issuecomment-1434823655 After my conversation with Andy, I removed the `-u`, documented my steps, and only bumped the bare minimum needed.